src/screens/DetailsPageScreens/DetailsPageScreen.tsx

```diff
@@ -42,47 +39,59 @@ const DetailsPageScreen: React.FC<DetailsPageScreenProps> = ({
   const [_, setSelectedTitle] = useState("");
   const [selectedTabItem, setSelectedTabItem] = useState("Nutrition");
 
+  const onSelectItem = () => {
+    navigate("/");
```

Comment:
AFAIK, we are currently not navigating users anywhere from the Detail page. Is there a reason to add `navigate()` here?

Reply:
This is for the go back button